pip-dep/requirements.txt

```diff
@@ -14,7 +14,7 @@ psutil==5.7.0
 requests~=2.22.0
 RestrictedPython~=5.0
 requests-toolbelt==0.9.1
-shaloop==0.2.1-alpha.11
+sycret==0.0.1-alpha.3
```

Comment:
Should we use the latests v0.0.1-alpha.6 ?

Reply:
Indeed, I moved to `sycret~=0.1.0` so we don't have to change the requirement for every new version -- unless we do an API-breaking change on the Sycret side :)